nucliadb_dataset/src/nucliadb_dataset/mapping.py

```diff
@@ -168,3 +169,41 @@ def func(batch: QuestionAnswerStreamingBatch, schema: pa.schema):
         return output_batch
 
     return func
+
+
+def batch_to_field_streaming_arrow():
+    def func(batch: FieldStreamingBatch, schema: pa.schema):
+        SPLIT = []
+        RID = []
+        FIELD = []
+        FIELD_TYPE = []
+        LABELS = []
+        EXTRACTED_TEXT = []
+        METADATA = []
+        BASIC = []
+        for data in batch.data:
+            SPLIT.append(data.split)
+            RID.append(data.rid)
+            FIELD.append(data.field)
+            FIELD_TYPE.append(data.field_type)
+            LABELS.append([x for x in data.labels])
+            EXTRACTED_TEXT.append(data.text.SerializeToString().decode())
+            METADATA.append(data.metadata.SerializeToString().decode())
+            BASIC.append(data.basic.SerializeToString().decode())
+        if len(RID):
+            pa_data = [
+                pa.array(SPLIT),
+                pa.array(RID),
+                pa.array(FIELD),
+                pa.array(FIELD_TYPE),
+                pa.array(LABELS),
+                pa.array(EXTRACTED_TEXT),
+                pa.array(METADATA),
+                pa.array(BASIC),
```

Comment:
this has different order than the one defined below (basic and metadata are swapped) is it a problem?